web/src/pages/apd/new/ApdNew.js

```diff
@@ -0,0 +1,210 @@
+import PropType from 'prop-types';
+import React, { Fragment, useState } from 'react';
+import { useForm, Controller } from 'react-hook-form';
+import { connect } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+
+import { Alert, Button, ChoiceList, TextField } from '@cmsgov/design-system';
+import { createApd } from '../../../redux/actions/app';
+
+import HitechView from './HitechView';
+import MmisView from './MmisView';
+
+import Joi from 'joi';
+
+const createApdSchema = Joi.object({
+  apdType: Joi.valid('hitech', 'mmis').required(),
+  name: Joi.string().min(1).required(),
+  years: Joi.array().min(3).required()
+});
+
+const ApdNew = () => {
+  ApdNew.displayName = 'ApdNew';
+  const history = useHistory();
+  const [apdType, setApdType] = useState(false);
+  const [{ createApd: create }] = useState(false);
```

Comment:
you can't use useState like this, I mean I guess you can't but you have no way to update it, you shouldn't be updated the createApd value directly